core/src/header_version/mod.rs

```diff
@@ -0,0 +1,12 @@
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+#[cfg(feature = "runtime")]
+use sp_runtime_interface::pass_by::PassByCodec;
+
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Encode, Decode, TypeInfo)]
+#[cfg_attr(feature = "runtime", derive(PassByCodec))]
+pub enum HeaderVersion {
+	V1, // Current one
+	V2, // To be used after runtime upgrade (new data_root)
```

Comment:
This should be the current one

Reply:
Since it was not used yet, did not want to introduce confusion